src/Launcher.cpp

```diff
@@ -155,159 +176,251 @@ int Launcher::automaticMerge() {
+            tr("The level of logging output you want to recieve."),
+            "verbose|debug");
+    parser.addOption(logLevelOpt);
+
+    QCommandLineOption noAlignOpt("no-align",
+            tr("Do not auto-align source images."));
+    parser.addOption(noAlignOpt);
+
+    QCommandLineOption noCropOpt("no-crop",
+            tr("Do not crop the output image to the optimum size."));
+    parser.addOption(noCropOpt);
+
+    QCommandLineOption groupOpt({"G", "group"},
+            tr("Determines how the input images are grouped.") +
+               "\n- all, a: " + tr("Process all input images as a single group. Default.") +
+               "\n- auto, t: " + tr("Group images automatically based on their time of creation (Exif DateTimeOriginal).") +
+               "\n- manual, m: " + tr("Group images manually based on a number of images per group."),
+            "all|auto|manual",
+            "all");
+    parser.addOption(groupOpt);
+
+    QCommandLineOption bracketSizeOpt({"s", "bracket-size"},
+            tr("Fixed number of images per bracket set. "
+               "Only used when grouping manually. " 
+               "Creation time and --single option will be ignored. "
+               "The total number of images passed to HDRMerge must be a multiple of this number.") +
+               "Default: 3",
+            "integer", QString::number(3));
+    parser.addOption(bracketSizeOpt);
+
+    QCommandLineOption gapOpt({"g", "gap"},
+            tr("Maximum gap in seconds between chronologically adjacent images. "
+               "If the gap is larger, a new group is crearted. "
+               "Only used when grouping automatically or all.") +
+               "Default: 3.",
+            "double", QString::number(3.0));
+    parser.addOption(gapOpt);
+
+    QCommandLineOption singleOpt("single",
+            tr("Include single images when specified. Only used when grouping automatically. "
+               "Off by default."));
+    parser.addOption(singleOpt);
+
+    QCommandLineOption helpOpt = parser.addHelpOption();
+
+    QCommandLineOption bitOpt("b",
+            tr("Bits per sample."),
+            "16|24|32", QString::number(16));
+    parser.addOption(bitOpt);
+
+    QCommandLineOption whiteLevelOpt("w",
+            tr("Use custom white level."), "integer", "16383");
+    parser.addOption(whiteLevelOpt);
+
+    QCommandLineOption featherRadiusOpt("r",
+            tr("Mask blur radius, to soften transitions between images. Default is 3 pixels."),
+            "integer", QString::number(3));
+    parser.addOption(featherRadiusOpt);
+
+    QCommandLineOption previewSizeOpt("p",
+            tr("Size of the preview. Default is none."),
+            "full|half|none", "none");
+    parser.addOption(previewSizeOpt);
+
+    parser.process(QCoreApplication::arguments());
+
+    bool ok; // to check conversions from QString
+
+    char const * errorStart = ">>> ERROR >>> ERROR >>> ERROR >>>\n";
```

Comment:
More `const`, please. :wink:

Reply:
OK. I will also move the const qualifier in front of the type to be more compliant with the rest of the codebase.

Reply:
Yes, please. :+1: